Automattic/wp-calypso
# e2e: Change how the node config is passed

#### Changes proposed in this Pull Request

* The `NODE_CONFIG` command line flag isn't work like it used to. I am setting it as an env variable instead in run.sh
* I also added a check to make sure that if the `LIVEBRANCHES` variable is set, like it is in the e2e test runs, that the URL contains calypso.live

#### Testing instructions
* Make sure tests pass in CI
* Run `Sign up for an account only (no site) then add a site` test locally with the command `./run.sh -g -s mobile -S fbbb85c9693e4b4513cda956aa9aaedab0f9ebf2` and make sure that the url that the test is running against is not  wordpress.com
